### DISCUSSION TITLE
GIMP packaging need help

### DISCUSSION THREAD

**Discussion by efa** [AppImage/AppImageKit#1246] [Q&A]
hi,
GIMP devs asked for help to generate an AppImage.
We as users hope this happen as current flatpack is massive, see:
https://gitlab.gnome.org/GNOME/gimp/-/issues/7661

**Comment by probonopd** [comment]
@aferrero2707 and @ivan-hc are probably most knowledgable about this, as they have made GIMP AppImages:

* https://github.com/aferrero2707/gimp-appimage
* https://github.com/ivan-hc/GIMP-appimage

It the GIMP team has any specific questions, I am happy to answer/help.

**Comment by efa** [comment]
aferrero repo is 3 year old.
ivan-hc does not work on my Debian, see: https://github.com/ivan-hc/GIMP-appimage/issues/6

**Comment by efa** [comment]
now ivan-hc's GIMP work on my Debian

**Comment by TheAssassin** [comment]
Converting to a discussion. @probonopd please move questions there, too.
